phpunit.xml

```diff
@@ -3,7 +3,6 @@
     xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
     bootstrap="tests/bootstrap.php"
     colors="true"
-    verbose="true"
```

Comment:
In case you want to stick with verbose mode: the upstream project was fixed and the next version of the gh-action should work then